descriptors/ModuleDescriptor-template.json

```diff
@@ -74,7 +74,7 @@
     },
     {
       "id": "circulation",
-      "version": "8.3",
+      "version": "8.4",
```

Comment:
In recent times, new APIs in circulation have typically been split into separate interfaces, to keep the interfaces small, allowing for more flexibility in what clients are dependent upon and what a module chooses to implement.

I think this endpoint is a candidate for being split out, as the clients that want to declare a loaned item lost are likely different to those who need check in or check out.

Reply:
This has been reverted to 8.3